src/ape/contracts/base.py

```diff
@@ -113,9 +122,62 @@ def __repr__(self) -> str:
         abis = sorted(self.abis, key=lambda abi: len(abi.inputs or []))
         return abis[-1].signature
 
+    def encode_input(self, *args) -> HexBytes:
+        args = self._convert_tuple(args)
+        selected_abi = _select_method_abi(self.abis, args)
```

Comment:
Can this be made into a separate method?

Reply:
What do you mean? I am interpreting that you mean make `select_method_abi()` a public method here for users but am not 100% sure!